.gitignore

```diff
@@ -4,3 +4,5 @@ packages/
 TestResults/
 *.suo
 *.nupkg
+.vs/
+*.user
```

Comment:
http://stackoverflow.com/questions/31878901/should-i-add-the-visual-studio-2015-vs-folder-to-source-control